include/myst/sockdev.h

```diff
@@ -163,4 +163,14 @@ struct myst_sockdev
 
 myst_sockdev_t* myst_sockdev_get(void);
 
+myst_sockdev_t* myst_sockdev_get_uds(void);
```

Comment:
Suggest renaming to `myst_udsdev_get` or `myst_sockdev_uds_get`.

Reply:
Good idea. I went with ``myst_udsdev_get()``.